ph5/core/ph5api.py

```diff
@@ -1283,7 +1283,11 @@ def cut(self, das, start_fepoch, stop_fepoch, chan=1,
                     das_t.append(d)
         # adjust the number of data samples as to not over extend the
         # cut_stop_fepoch
-        calc_stop_fepoch = trace_start_fepoch + (len(data) / sr)
+        if data is None:
```

Comment:
You're duplicating https://github.com/PIC-IRIS/PH5/blob/9a7d7657e28e5f880fd2be12a32bb0300401e89f/ph5/core/ph5api.py#L1300
Should it be just moved up here, and deleted below?

Reply:
The second `if data is None: ` was the original check and was left to make minimal changes. It seems okay to just move it and delete.

Reply:
The duplicated block and the else statement have been removed by commit 0e453b2d464ec0ee66d50df36919d1d70872c98e. Let me know if I should change anything else.